source/install/config-cloudfront.rst

```diff
@@ -0,0 +1,40 @@
+.. _config-cloudfront:
+
+Configuring CloudFront to host Mattermost static assets
+=======================================================
+
+1. Create an S3 bucket using your desired domain. In our example it will be mattermost.example.com.
+2. Enable static hosting for you S3 bucket.
+3. From the Mattermost distribution, upload the ``client`` directory to s3. Renaming it to ``static``. You can use aws CLI command below from within the client directory. Be sure however you do this that the files are public readable.
+
+    ``aws s3 cp --acl public-read --recursive . s3://static.spinmint.com/static/``
+
+4. Setup your Mattermost app server and create a record from a subdomain of your desired domain to point directly to your app server. This is to bypass CloudFront to connect websockets. For our example we will use the domain ws.mattermost.example.com. If you have multiple app servers, this domain should point to the load balancer/proxy such as ALB or nginx.
```

Comment:
ALB or ELB?

Reply:
ALB actually. Amazon Application Load Balancer. The newer version of ELB.